packages/server/lib/project-base.ts

```diff
@@ -160,19 +160,12 @@ export class ProjectBase extends EE {
 
     this._server = new ServerBase(cfg)
 
-    let studioManager: StudioManager | null
-
-    if (process.env.CYPRESS_ENABLE_CLOUD_STUDIO || process.env.CYPRESS_LOCAL_STUDIO_PATH) {
-      studioManager = await getAndInitializeStudioManager({
-        projectId: cfg.projectId,
-        cloudDataSource: this.ctx.cloud,
-      })
-
-      this.ctx.update((data) => {
-        data.studio = studioManager
-      })
-
-      if (studioManager.status === 'INITIALIZED') {
+    // Start studio manager initialization but don't block
+    const studioManagerPromise = getAndInitializeStudioManager({
```

Comment:
I'm wondering whether we could move this promise to be either part of the constructor for StudioLifecycleManager (so that it's not really mutable after its set). Or maybe even better, encapsulate this logic entirely inside of StudioLifecycleManager (but only if it's not super messy to do that).

Reply:
I updated lifecycle manager so that it handles all of the initialization stuff internally - so all project base does is call initialize and then registers its listener. I think this is better